index.js

```diff
@@ -1,36 +1,47 @@
 var path = require("path");
 const express = require("express");
+const bodyParser = require("body-parser");
 const app = express();
 const port = process.env.PORT || 3000;
-import * as dotenv from "dotenv";
-dotenv.config();
+require("dotenv").config();
 
 app.use("/static", express.static(path.join(__dirname, "public")));
+app.use(bodyParser.urlencoded({ extended: true }));
+
 app.engine(".html", require("ejs").__express);
 app.set("views", path.join(__dirname, "views"));
 app.set("view engine", "html");
 
 const { Client } = require("pg");
-
 const client = new Client({
   connectionString: process.env.DATABASE_URL,
   ssl: {
     rejectUnauthorized: false,
   },
 });
 
-client.connect();
+// username: string - the user name of the user
+// onSuccess: function - the function that will be called when the query is successful.
+// onError: function - the function that will be called when the query is errored.
+function getUser(email, onSuccess, onError) {
```

Comment:
This function does a database query and then calls one of 2 functions that are pass in as parameters. If the query is successful, it executes `onSuccess`. If it has an error, it calls `onError`.